sql/core/src/test/resources/tpcds-query-results/v1_4/q17.sql.out

```diff
@@ -0,0 +1,6 @@
+-- Automatically generated by TPCDSQueryTestSuite
+
+-- !query schema
+struct<i_item_id:string,i_item_desc:string,s_state:string,store_sales_quantitycount:bigint,store_sales_quantityave:double,store_sales_quantitystdev:double,store_sales_quantitycov:double,as_store_returns_quantitycount:bigint,as_store_returns_quantityave:double,as_store_returns_quantitystdev:double,store_returns_quantitycov:double,catalog_sales_quantitycount:bigint,catalog_sales_quantityave:double,catalog_sales_quantitystdev:double,catalog_sales_quantitycov:double>
+-- !query output
+
```

Comment:
This should be 
```
+AAAAAAAAKPFEAAAA       Recently right  TN      1       99.0    NULL    NULL    1       66.0    NULL    NULL    1       32.0    NULL    NULL
```
A PR to fix this bug is: https://github.com/apache/spark/pull/31940

Reply:
This seems to be caused by the wrong schema of `date_dim`: https://github.com/apache/spark/pull/31012/files#r599447123

Reply:
Fixed in #31012